src/coreclr/vm/stubmgr.cpp

```diff
@@ -2104,18 +2127,30 @@ BOOL InteropDispatchStubManager::TraceManager(Thread *thread,
 
     if (IsVarargPInvokeStub(GetIP(pContext)))
     {
-        NDirectMethodDesc *pNMD = (NDirectMethodDesc *)arg;
-        _ASSERTE(pNMD->IsNDirect());
-        PCODE target = (PCODE)pNMD->GetNDirectTarget();
+        #if defined(TARGET_ARM64) && defined(__APPLE__)
+            //On ARM64 Mac, we cannot put a breakpoint inside of VarargPInvokeStub
+            LOG((LF_CORDB, LL_INFO10000, "IDSM::TraceManager: Skipping on M1\n"));
+            return FALSE;
+        #else
+            NDirectMethodDesc *pNMD = (NDirectMethodDesc *)arg;
+            _ASSERTE(pNMD->IsNDirect());
+            PCODE target = (PCODE)pNMD->GetNDirectTarget();
 
-        LOG((LF_CORDB, LL_INFO10000, "IDSM::TraceManager: Vararg P/Invoke case 0x%p\n", target));
-        trace->InitForUnmanaged(target);
+            LOG((LF_CORDB, LL_INFO10000, "IDSM::TraceManager: Vararg P/Invoke case 0x%p\n", target));
```

Comment:
```suggestion
            LOG((LF_CORDB, LL_INFO10000, "IDSM::TraceManager: Vararg P/Invoke case %p\n", target));
```

The `0x` isn't needed for `%p`. This was already there, but should be changed.